examples/07-routing/src/app.gleam

```diff
@@ -94,36 +97,18 @@ fn update(model: Model, msg: Msg) -> #(Model, Effect(Msg)) {
 fn view(model: Model) -> Element(Msg) {
   let styles = [#("margin", "15vh")]
 
-  let page = case model.current {
-    WelcomeGuest(name) -> render_welcome(model, name)
+  let page = case model.current_route {
     Home -> render_home(model)
+    WelcomeGuest(name) -> render_welcome(model, name)
   }
 
   ui.stack([attribute.style(styles)], [render_nav(model), page])
 }
 
-fn render_nav(model: Model) -> Element(a) {
-  let item_styles = [#("margin", "15px"), #("text-decoration", "underline")]
-
-  let nav_item = fn(path, text) {
-    html.a([attribute.href("/" <> path), attribute.style(item_styles)], [
-      element.text(text),
-    ])
-  }
-
-  let guests =
-    model.guests
-    |> list.map(fn(guest: Guest) {
-      nav_item("welcome/" <> guest.slug, guest.name)
-    })
-
-  html.nav([], [nav_item("", "Home"), ..guests])
-}
-
 fn render_home(model: Model) {
```

Comment:
In Lustre we call these _view functions_, not _render_: could you rename these all to `view_x`?